.devcontainer/docker-compose.override.yaml

```diff
@@ -9,6 +9,10 @@ services:
       target: "dev_image"
     volumes:
       -  database_volume:/mzai/backend/local.db
+    ports:
+      - "5678:5678"
+    environment:
+      - DEBUG=1
```

Comment:
Would you consider passing this as a flag in the Makefile? Typically, when building something, you indicate whether it’s a release or a debug build through the build tool. I know Python isn’t like C++, but what do you think?